src/content/en/updates/2017/02/lighthouse-jan.md

```diff
@@ -0,0 +1,150 @@
+that were used to create the report; a [long time feature request](https://github.com/GoogleChrome/lighthouse/issues/568):
+
+![Emulation settings](../../images/2017/02/lighthouse/image_5.gif)
+
+### More useful trace data
+
+Lighthouse metrics like "First meaningful paint", "Time to Interactive", etc are
+mocked out as User Timing measures and injected back into the trace data saved with the `--save-assets` flag.
+
+If you use the  `--save-assets` flag when running Lighthouse, drop the trace into
+the DevTools or open it in the [DevTools Timeline Viewer]([https://chromedevtools.github.io/timeline-viewer/](https://chromedevtools.github.io/timeline-viewer/)).
+You'll be able to see your key metrics in context with the full trace of the page load.
+
+![Trace data](../../images/2017/02/lighthouse/image_6.png)
+
+## Extension
+
+Extension users may notice Chrome warning about a change in permissions. You may need to re-enable the Lighthouse extension after Chrome auto-updates it.
+{: .note }
+
+One difference between Lighthouse runs on the command line versus in the
+extension is the fact that the CLI version can disable many more features of
+Chrome that may interfere with measuring performance, not the least of which
+are other extensions.
+
+In an attempt to make Lighthouse more consistent everywhere it's used, the
+extension's settings now has an option to disabled other Chrome extensions
+before running Lighthouse. If we have problems auditing your page, try enabling
+this option to see if it fixes the issue.
+
+![Disable other extensions setting](../../images/2017/02/lighthouse/image_7.png)
+
+## Lighthouse Viewer
+
+In HTML reports, you'll notice a new button with options for exporting the
+report in different formats. One of those options is "Open in Viewer". Clicking
+this button will send the report to the online [Viewer]([https://googlechrome.github.io/lighthouse/viewer/](https://googlechrome.github.io/lighthouse/viewer/)),
+where you can further share the report with users using Github.
+
+![Open in Viewer button](../../images/2017/02/lighthouse/image_8.png)
+
+![Open in Viewer result](../../images/2017/02/lighthouse/image_9.png)
+
+Behind the scenes, Viewer gets your permission via OAuth to create a GitHub
+[secret gist](https://help.github.com/articles/about-gists/#secret-gists) and
+saves the report there. Since it's done as your Gist, you maintain full control
+over the sharing of the report and you can delete it at any time. You can revoke
+the Viewer's permission to create gists under your [GitHub settings](https://github.com/settings/applications).
+
+## Performance Experiment
+
+The first version of the [Performance Experiment](https://github.com/GoogleChrome/lighthouse/issues/1143)
+project has landed in 1.5.0. This lets you experiment with your page load performance,
+interactively testing the effects of blocking or delaying assets in your critical
+path **during development**.
+
```

Comment:
Is there value in linking up to https://docs.google.com/document/d/1FYt5Es_Kf5IyC_bkTHj2G_a_sTvRvIq5iZCEN8VZY5o/edit#heading=h.cetla8h0y4o for folks that don't know what the feature does behind the scenes?